terraform/modules/secret_manager/main.tf

```diff
@@ -0,0 +1,48 @@
+resource "google_project_service" "services" {
+  project = var.project_id
+
+  service                    = "secretmanager.googleapis.com"
+  disable_on_destroy         = false
+  disable_dependent_services = false
+}
+
+# Secret Manager secrets for jira plugin to use.
+resource "google_secret_manager_secret" "jira_api_token" {
+  project = var.project_id
+
+  secret_id = var.secret_id
+  labels    = var.labels
+  replication {
+    automatic = true
+  }
+
+  depends_on = [
+    google_project_service.services
+  ]
+}
+
+resource "google_secret_manager_secret_version" "jira_api_token_version" {
+  secret = google_secret_manager_secret.jira_api_token.id
+
+  # default value used for initial revision to allow cloud run to map the secret
+  # to manage this value and versions, use the google cloud web application
+  secret_data = "DEFAULT_VALUE"
```

Comment:
Ignore change on this as well? Otherwise, reapply this terraform will wipe out the data and set to "DEFAULT_VALUE" everytime.

Reply:
Is the plan to have somebody manually save the JIRA cred in the secret after it's provisioned?

Reply:
Good catch @sqin2019 , added `secret_data` to the ignore list

Re @yolocs: Yeah, that is what I am thinking. Plain text cannot be included in the PR, so having a terraform script to set secret_data doesn't seem necessary. Let me know if you have other ideas